server/block/sugar_cane.go

```diff
@@ -0,0 +1,101 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+)
+
+// SugarCane is a plant block that generates naturally near water.
+type SugarCane struct {
```

Comment:
SugarCane should implement LiquidRemovable

Reply:
server/block/crop.go:40:15: has another declaration of HasLiquidDrops.

Reply:
SugarCane is not embedding crop, nor should it

Reply:
ahh, found my mistake. added the func now.